src/main.sh

```diff
@@ -714,13 +714,21 @@ install_package() {
 	for pkg in ${pkgname[@]}; do
 		fullver=$(NOEPOCH=1 get_full_version)
 		pkgarch=$(get_pkg_arch $pkg)
-		pkglist+=("${PKGDEST}/${pkg}_${fullver}_${pkgarch}.deb")
+		tmpfile="$(mktemp "/tmp/${pkg}_${fullver}_${pkgarch}_XXXXXX.deb")"
+		cp "${PKGDEST}/${pkg}_${fullver}_${pkgarch}.deb" "${tmpfile}"
+		pkglist+=("${tmpfile}")
 	done
 
 	if ! SUDO=1 makedeb_rs install "${APTARGS[@]}" "${pkglist[@]}"; then
 		warning "$(gettext "Failed to install built package(s).")"
 		return $E_INSTALL_FAILED
 	fi
+	for pkg in ${pkglist[@]}; do
+		if ! rm "${pkg}"; then
+      error "$(gettext "Failed to remove temporary package file.")"
+      return $E_INSTALL_FAILED
+    fi
```

Comment:
This indentation just needs fixed up (it happens to me in Neovim all the time, idk why :P), but other than that it all looks good! No clue why the tests aren't running, I'll try to get that fixed up though. Looks good at first glance though :).

Reply:
@hwittenborn  Good find. If you `:set noexpandtab`, then that should fix the issue. I added a modeline at the bottom so that this shouldn't happen anymore, at least in this file.

Since NeoVim 0.9, I believe you can just add an `.editorconfig` file so that this would be project-wide, but I'm on my phone at the moment and the modeline was easier to add.

Reply:
Oh cool, I didn't know Neovim could do that. It's fine just having that indentation stuff in that file for now - the project currently has some pretty wacky indentation styles - some use 4-space indents (which is what I'm wanting), others use tabs, and others are some kind of weird hybrid.

What you have right now is fine though - I'm not really focused on doing any super major refactorings in makedeb for the time being since I'm starting on this new tool, and I don't think anything super major is going to happen in the close future.